asm-patch/Kptfile

```diff
@@ -93,6 +93,11 @@ openAPI:
         setter:
           name: anthos.servicemesh.tag
           value: 1.6.8-asm.9
+    io.k8s.cli.setters.anthos.servicemesh.canonicalServiceHub:
+      x-k8s-cli:
+        setter:
+          name: anthos.servicemesh.canonicalServiceHub
+          value: gcr.io/gke-release/asm/canonical-service-controller:1.6.8-asm.9
```

Comment:
will we update the canonical-service-controller tag to 1.7.1-asm.x?

Reply:
I'd like to, but that doesn't seem to have happened yet.  Right now this is the only available version tag.  We can absolutely update it when it's available, but functionally it would be a no-op.